js/components/Table/Row.js

```diff
@@ -6,19 +6,28 @@ class Row extends React.Component {
     super();
   }
 
-  getImage(data) {
-    return 'images/college_logos/' + data.northstar_id +'.png';
+  getImage(northstar_id) {
+    return 'images/college_logos/' + northstar_id +'.png';
+  }
+
+  getCollegeName(northstar_id) {
+    const collegeNames = {
+      // 'northstar_id_here': 'Not Real University',
+    };
+
+    return collegeNames[northstar_id];
   }
```

Comment:
awesome!! Will this break if it is commented out right now?

Reply:
@chloealee great question, I made sure that it still worked like this!